docs/documentation/data.xml

```diff
@@ -0,0 +1,17 @@
+<FASTOAD_model>
+  <data>
+    <geometry>
+      <wing>
+        <area units="m**2">150.0</area>
+      </wing>
+    </geometry>
+    <weight>
+      <fuselage>
+        <mass units="kg">10000.0</mass>
+        <CG>
+          <x units="m">20.0</x>
+        </CG>
+      </fuselage>
+    </weight>
+  </data>
+</FASTOAD_model>
```

Comment:
Is this file necessary ? Since in the documentation it is shown as a code-block and not as reading this file (which I don't know if it is possible)